tests/cephadm/test_admin_node_removal.py

```diff
@@ -0,0 +1,33 @@
+from ceph.ceph_admin import CephAdmin
+from utility.log import Log
+
+log = Log(__name__)
+
+
+def run(ceph_cluster, **kw):
+    """Verify removal of _admin node acting as a bootstrap node
+    Args:
+        **kw: Key/value pairs of configuration information to be used in the test.
+    Returns:
+        0 - if test case pass
+        1 - it test case fails
+    Test case covered : CEPH-83574670
+    Test Case Flow:
+    1. Deploy a cluster and check for admin node
+    2. Try to remove the _admin node from bootstrap node
+    """
+
+    try:
```

Comment:
Same as above.

Reply:
Thanks @vamahaja
Removed

Reply:
@MohitBis except block handles very general exception. It will return 0 even in case it fails to get configs. I suggest to use specific exception and check error messages for required massages.

Reply:
@vamahaja 
Added try and exception block for command execution only.